Content.Shared/Emoting/EmoteEvents.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Content.Shared.Chat.Prototypes;
+using Content.Shared.Inventory;
+
+namespace Content.Shared.Emoting
+{
+    public sealed class EmoteAttemptEvent : CancellableEntityEventArgs
+    {
+        public EmoteAttemptEvent(EntityUid uid)
+        {
+            Uid = uid;
+        }
+
+        public EntityUid Uid { get; }
+    }
+}
+
+/// <summary>
+/// An event which is raised to determine if the entity's ability to perform certain emotes is blocked
+/// </summary>
+[ByRefEvent]
+public record struct GetEmoteBlockersEvent() : IInventoryRelayEvent
```

Comment:
This event has to be in `Content.Shared`, and I saw some other examples of multiple events being put in the same file. Lemme know if this is undesireable.